src/app/zap-templates/zcl/provisioning.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+    <domain name="CHIP" spec="chip-0.7" dependsOn="zcl-1.0-07-5123-03" certifiable="true"/>
+    <enum name="NetworkProvisioningError" type="ENUM8">
+        <item name="Success" value="0x0"/>
+        <item name="OutOfRange" value="0x1"/>
+        <item name="BoundsExceeded" value="0x2"/>
+        <item name="NetworkIDNotFound" value="0x3"/>
+        <item name="DuplicateNetworkID" value="0x4"/>
+        <item name="NetworkNotFound" value="0x5"/>
+        <item name="RegulatoryError" value="0x6"/>
+        <item name="AuthFailure" value="0x7"/>
+        <item name="UnsupportedSecurity" value="0x8"/>
+        <item name="OtherConnectionFailure" value="0x9"/>
+        <item name="IPV6Failed" value="0xa"/>
+        <item name="IPBindFailed" value="0xb"/>
+        <item name="Label9" value="0xc"/>
+        <item name="Label10" value="0xd"/>
+        <item name="Label11" value="0xe"/>
+        <item name="Label12" value="0xf"/>
+        <item name="Label13" value="0x10"/>
+        <item name="Label14" value="0x11"/>
+        <item name="Label15" value="0x12"/>
```

Comment:
Do we really want those `Label**` strings here ?

Reply:
Just follows the spec, we will update this file as the spec is updating.